### ISSUE TITLE
wrap-with-body logger missing timbre

### ISSUE THREAD

**Issue by fzakaria** [nberger/ring-logger#10]
Can you give a better idea on how to call wrap-with-body-logger if we are using timbre.
The function definition seems to be different than that of wrap-with-logger which I'm not sure why.

**Comment by nberger** [comment]
When using timbre, wrap-with-body-logger would be used as following:

``` clojure
(require '[ring.logger.timbre :refer [make-timbre-logger]])

(def handler (-> app
                 wrap-with-logger
                 (wrap-with-body-logger (make-timbre-logger)))
```

`wrap-with-body-logger` takes just the handler and the logger, because there are no additional options like in `wrap-with-logger`.

Btw, I just checked in 0.7.2 and wrap-with-body-logger is not working at all. Will be releasing ring-logger 0.7.3 with a fix soon.

**Comment by nberger** [comment]
Just released `[ring-logger "0.7.4"]` and `[ring-logger-timbre "0.7.4"]` including some bugfixes and adding `logger.timbre/wrap-with-body-logger` so you can now do:

``` clojure
(require '[ring.logger.timbre :as logger.timbre])

(def handler (-> app
                 logger.timbre/wrap-with-logger
                 logger.timbre/wrap-with-body-logger))
```

**Comment by fzakaria** [comment]
Wicked.
Thanks for the quick fix.
